aiu_fms_testing_utils/__init__.py

```diff
@@ -0,0 +1 @@
+from ._version import __version__ 
```

Comment:
@tharapalanivel @JRosenkranz @ani300 This change allow us to get the version from the setuptool-scm generated _version.py file.
For example:
python -c "import aiu_fms_testing_utils;print(aiu_fms_testing_utils.__version__)"

Reply:
@jbusche this line failed lint check. I tested this with `ruff check .`
```suggestion
from ._version import __version__

__all__ = ['__version__']
```
python -c "import aiu_fms_testing_utils;print(aiu_fms_testing_utils.__version__)"